ui/src/app/cron-workflows/components/schedule-validator.tsx

```diff
@@ -5,6 +5,9 @@ const x = require('cronstrue');
 
 export const ScheduleValidator = ({schedule}: {schedule: string}) => {
     try {
+        if (schedule.split(' ').length === 6) {
```

Comment:
Also provide an error message when length > 6 so users can be aware of mistakes earlier in the UI?